README.md

```diff
@@ -60,6 +61,7 @@ country | no | '' | string | This tells the component what country to grab the l
 defaultRegion | no | 'US' | string | This allows you to set a default region when choosing not to use the country attribute. It will be set to regions of the United States if not provided.
 countryName | no | false | boolean | Set this to true if you are setting it to true while using Country Select. This is just to help keep the data values in sync.
 regionName | no | false | boolean | Set this to true if you want the v-model to output full region names instead of the default abbreviations.
+class | no | [] | string | Class name ex: `form-control` 
```

Comment:
change class to className as class is a javascript keyword. Also change [] to '' as default value is string not array.